### PR TITLE
Python formatting tools

### PR DESCRIPTION
**Pull request by Xierumeng**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    "Run",
+    "_",
+
+    # Return of main()
+    "result_main",
+]
+
+# Good variable names regexes, separated by a comma. If names match any regex,
+# they will always be accepted
+good-names-rgxs = "computer-vision-python"
```
Comment (TongguangZhang) [inline pyproject.toml:48]: Remove

File: qr/test_qr.py [hunk 2]
```diff
@@ -15,38 +16,38 @@
...
+    + "Route number 11: 5 pers; Zulu; Oscar; 15 kg; No landing Zulu – Hover for 15s; $80\n"
+    + "Route number 12: 2 pers; Bravo; Papa; 10 kg; nil; $80\n"
+    + "Route number 13: 2 pers; Hotel; Zulu; 15 kg; No landing Zulu – Hover for 15s; $42\n"
+    + "Route number 14: 2 pers; Whiskey; Hotel; 15 kg; nil; $28\n"
+    + "Route number 15: 6 pers; Tango; Oscar; 15 kg; nil; $108\n"
+    + "Route number 16: 4 pers; Hotel; Tango; 15 kg; nil; $68\n"
+    + "Route number 17: 3 pers; Papa; Zulu; 15 kg; No landing Zulu – Hover for 15s; $93\n"
+    + "Route number 18: 2 pers; Bravo; Whiskey; 10 kg; nil; $35\n"
+    + "Route number 19: 2 pers; Whiskey; Zulu; 15 kg; No landing Zulu – Hover for 15s; $110\n"
+)
```
Comment (TongguangZhang) [inline qr/test_qr.py:42]: Use multiline string

**Review by TongguangZhang — COMMENTED** [review]
Reviewed

File: requirements.txt [hunk 3]
```diff
@@ -19,4 +19,9 @@ dronekit
 pyzbar
 
 # kml
```
Comment (TongguangZhang) [inline requirements.txt:21]: Remove commented out code

Comment (Xierumeng) [reply requirements.txt:17]: This is the name of the module.

**Review by TongguangZhang — APPROVED** [review]
LGTM!

**Pull request merged**